### PR TITLE
#154 gui.__frame and a bit of doc

### PR DESCRIPTION
**Pull request by FredLL-Avaiga**
PS: _set_frame for Florian V

### CODE REVIEW

File: gui/doc/chart.md [hunk 1]
```diff
@@ -158,7 +158,8 @@ Depending on the algorithm, Some specific points might not be shown on the chart
 
 ### Responsive chart
 
-By specifying _plot_config_ responsive key, chart will be responsive ie chart will resize automatically to adapt to its container width.
+By specifying _plot_config_ responsive key, chart will be responsive ie chart will resize automatically to adapt to its container width (especially usefull on mobile devices).
+[Plotly.js responsive](https://plotly.com/javascript/responsive-fluid-layout/)
```
Comment (FabienLelaquais) [inline gui/doc/chart.md]: Please use:
"See get more information at [Plotly.js responsive](https://plotly.com/javascript/responsive-fluid-layout/)."

**Review by FabienLelaquais — APPROVED** [review]
👍

**Review by dinhlongviolin1 — APPROVED** [review]
Looks good to me!

File: gui/doc/chart.md (hunk 1 above)
Comment (FredLL-Avaiga) [reply gui/doc/chart.md]: See get ?

Comment (FredLL-Avaiga) [reply gui/doc/chart.md]: I suppose you mean Get

Comment (FabienLelaquais) [reply gui/doc/chart.md]: Sorry... I meant 'Please get...' (and the final dot)

**Review by florian-vuillemot — APPROVED** [review]

**Pull request merged**